digitalpublicgoods/digit-public-finance-management.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "DIGIT Public Finance Management",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://ifix.digit.org/"
+  },
+  "platformIndependence": {
```

Comment:
They use Elastic License 2.0 which uses a license that is not OSI compliant. They responded that this is not an integral component of the solution:

> ElasticSearch is not mandatory but it is what we use as a datastore for analytics. Developers can use any open or closed source Analytics platform for storing the fiscal events emitted by iFIX. We use ElasticSearch as an example. 

@iperdomo please provide input, thanks.